packages/dendron-plugin-views/src/components/DendronLookupPanel.tsx

```diff
@@ -0,0 +1,233 @@
+import { DendronProps } from "../types";
+import { Form, Select, Switch, FormInstance } from "antd";
+import {
+  DMessageSource,
+  LookupEffectTypeEnum,
+  LookupFilterTypeEnum,
+  LookupNoteTypeEnum,
+  LookupSelectionTypeEnum,
+  LookupSplitTypeEnum,
+  LookupViewMessage,
+  LookupViewMessageEnum,
+} from "@dendronhq/common-all";
+import React, { useEffect } from "react";
+import { postVSCodeMessage } from "../utils/vscode";
+import { ideSlice } from "@dendronhq/common-frontend";
+
+const { Option } = Select;
+
+export default function DendronLookupPanel({ ide }: DendronProps) {
+  const [form] = Form.useForm();
+
+  // on each render
+  useEffect(() => {
+    // ask vscode for controller state if it's not set yet.
+    if (ide.lookupModifiers === undefined) {
+      postVSCodeMessage({
+        type: LookupViewMessageEnum.onRequestControllerState,
+        data: {},
+        source: DMessageSource.webClient,
+      } as LookupViewMessage);
+    }
+  });
+
+  return (
+    <>
+      <h4>Modifiers</h4>
+      <LookupViewForm ide={ide} form={form} />
+    </>
+  );
+}
+
+function LookupViewForm({
+  ide,
+  form,
+}: {
+  ide: ideSlice.IDEState;
+  form: FormInstance;
+}) {
+  const pressed = ide.lookupModifiers
+    ? ide.lookupModifiers.filter((mod) => {
+        return mod.pressed;
+      })
+    : [];
+
+  // update selection type form
+  const selectionTypeState = pressed.filter((mod) => {
+    if (Object.keys(LookupSelectionTypeEnum).includes(mod.type)) {
+      form.setFieldsValue({ selection: mod.type });
+      return true;
+    }
+    return false;
+  });
+
+  if (selectionTypeState.length === 0) {
+    form.setFieldsValue({ selection: undefined });
+  }
+
+  // update note type form
+  const noteTypeState = pressed.filter((mod) => {
+    if (Object.keys(LookupNoteTypeEnum).includes(mod.type)) {
+      form.setFieldsValue({ note: mod.type });
+      return true;
+    }
+    return false;
+  });
+
+  if (noteTypeState.length === 0) {
+    form.setFieldsValue({ note: undefined });
+  }
+
+  // update effect type form
+  form.setFieldsValue({
+    effect: pressed
+      .filter((mod) => {
+        return Object.keys(LookupEffectTypeEnum).includes(mod.type);
+      })
+      .map((mod) => mod.type),
+  });
+
+  // update horizontal split switch
+  form.setFieldsValue({
+    horizontalSplit:
+      pressed.filter((mod) => {
+        return Object.keys(LookupSplitTypeEnum).includes(mod.type);
+      }).length === 1,
+  });
+
+  // update direct child only switch
+  form.setFieldsValue({
+    directChildOnly:
+      pressed.filter((mod) => {
+        return Object.keys(LookupFilterTypeEnum).includes(mod.type);
+      }).length === 1,
+  });
+
+  return (
+    <>
+      <Form form={form}>
+        <SelectionTypeFormItem />
```

Comment:
can we have a tooltip when hovering over the options that have a 1 sentence description of what they do?